src/block/block_ext.c

```diff
@@ -33,12 +33,17 @@ static int __block_merge(WT_SESSION_IMPL *, WT_BLOCK *, WT_EXTLIST *, wt_off_t,
  *     Return the last element in the list, along with a stack for appending.
  */
 static WT_INLINE WT_EXT *
-__block_off_srch_last(WT_EXT **head, WT_EXT ***stack)
+__block_off_srch_last(WT_EXTLIST *el, WT_EXT ***stack, bool need_traverse)
 {
     WT_EXT **extp, *last;
+    WT_EXT **head;
```

Comment:
This line/variable should be merged with the line above, and the variables in alphabetical order.